universal_transfer_operator/src/universal_transfer_operator/data_providers/filesystem/aws/s3.py

```diff
@@ -123,41 +125,48 @@ def download_file(self, file) -> TempFile:
 
     @property
     def verify(self) -> str | bool | None:
-        return self.dataset.extra.get("verify", None)
+        verify: str | bool | None = self.dataset.extra.get("verify", None)
```

Comment:
What does this property represent? Would it be worth trying to normalise it always to be a boolean?